src/common/transformations/include/transformations/transpose_sinking/ts_binary.hpp

```diff
@@ -4,8 +4,7 @@
 
 #pragma once
 
-#include "openvino/pass/graph_rewrite.hpp"
-#include "openvino/pass/pass.hpp"
+#include "openvino/pass/MatcherPass.hpp"
```

Comment:
Maybe is possible to include some what is really required for header here and other parts move to cpp file when are required for implementation.

If  less dependencies provide for the header the others which use this transformation will also have less.
Maybe using forward declaration will help and put more includes inside cpp files with components which are used there.